css-layout-classes/styles.css

```diff
@@ -0,0 +1,28 @@
+body {
+  font-family: sans-serif;
+}
+
+.container {
+  max-width: 800px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-one-third {
+  flex-basis: calc(100% / 3);
+}
+
+.column-one-fourth {
+  flex-basis: calc(100% / 4);
+}
+
+.column-two-thirds {
+  flex-basis: 66%;
```

Comment:
Really small detail here. It's totally fine to have it as 66% but `calc` can be used here with "/ 3 * 2" so that it gives pixel-perfect calculation